create.go

```diff
@@ -81,7 +81,12 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 	}
 
 	// Start the shim to retrieve its PID.
-	pid, err := startShim(pod)
+	containers := pod.GetContainers()
+	if len(containers) != 1 {
+		return fmt.Errorf("Container list from pod is wrong, expecting only one container")
```

Comment:
This indicates a programming error, so I'd prefix with `BUG:` and maybe show how many containers you actually found.